process-compose.yaml

```diff
@@ -66,6 +67,18 @@ processes:
         port: $ESPRESSO_CONSENSUS_SERVER_PORT
         path: /healthcheck
 
+  state-relay-server:
+    command: state-relay-server
+    environment:
+      - ESPRESSO_STATE_RELAY_SERVER_PORT:$ESPRESSO_STATE_RELAY_SERVER_PORT
+      - ESPRESSO_STATE_SIGNATURE_WEIGHT_THRESHOLD:$ESPRESSO_STATE_SIGNATURE_WEIGHT_THRESHOLD
```

Comment:
If we're just passing through the value from the environment I think you can just put its name, no need to give the value